packages/window-size-tool/rules.json

```
@@ -1,52 +1,57 @@
 [
   {
     "width": "< 1200",
-    "terms": ["//CLUSTER/@NUM_COLS > 4", "count(//LIST/*) > 6"],
+    "terms": ["//CLUSTER/@NUM_COLS > 4", "count(//LIST[count(FIELD | CONTAINER) > 6 ]) > 0"
+    ],
     "target": "xlg"
   },
   {
-    "width": ">= 768",
+    "width": "<= 768",
     "terms": [
       "//CLUSTER/@NUM_COLS >= 3",
-      "count(//CLUSTER[@NUM_COLS >= 2]//LIST) > 0",
-      "count(//LIST/*) = 5 or count(//LIST/*) = 6"
+      "count(//CLUSTER[@NUM_COLS > 2]//LIST) > 0",
```

Keep the `=` because we want this rule to apply if there is a CLUSTER with 2 or more columns _and_ it contains a LIST.

If it doesn't contain a list, then this rule will be ignored.

In that scenario if the modal is >576 it will not be modified, just staying the same size.

If it is <= 576 (and doesn't contain a list) then the rule on [line 28](https://github.com/IBM/spm-ui-upgrade-helper/pull/170/files#diff-06a843c6ab44a689e160f03eff83ab1e0e5caf377803515c1b87bd937f7f32a6R28) will apply.

```suggestion
      "count(//CLUSTER[@NUM_COLS >= 2]//LIST) > 0",
```